config/plugins/tours/core.history.yaml

```diff
@@ -1,3 +1,4 @@
+id: galaxy_history
```

Comment:
Can you explain adding the id to the tours here? Does this get propagated somehow? It looks like the id was calculated in a different way before?

Reply:
The id is calculated based on the filename [here](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tours/__init__.py#L73), so the id from the yaml file is not used at all for tour loading. I checked the client ([client/src/mvc/tours.js](https://github.com/galaxyproject/galaxy/blob/dev/client/src/mvc/tours.js#L43) appears to be the only relevant file) - same there. I considered deleting the id field, but instead decided to add id values to the other 2 tours for consistency, in case there may have been plans for that field. I don't mind either deleting the field or reverting my edit and leaving as is, but having that field optional (given that we formalize it in a pydantic model) doesn't seem right, I think.

Reply:
Thanks for looking into that - so it seems it isn't used in a functional capacity anywhere? Want to remove it from core.galaxy_ui.yaml and see if anyone complains? I think we should remove it if it is not being used.

Reply:
Done! Completely agree on removing dead code.